### DISCUSSION TITLE
Avoid duplicate when relaunching template

### DISCUSSION THREAD

**Discussion by luccomo** [SilentVoid13/Templater#1083] [Help]
I have part of my Daily Notes a Create Daily Work Notes
see here
Work Daily Notes: `[[<% (await tp.file.create_new("#work #WorkDaily","WorkDaily - "+tp.file.creation_date("MM-DD-YYYY"))).basename %>]]`

but if i click again on my Daily Notes template, it will Create another new file and add "1" increment as link.
How can i say - if the Create New exists just keep the link to existing one
Thanks all

**Comment by Zachatoo** [comment]
I have a few examples of how to do that [here](https://zachyoung.dev/snippets/templater#create-file-if-it-doesnt-exist). Here's a specific answer for yours:

```js
<%*
const fileName = "WorkDaily - " + tp.file.creation_date("MM-DD-YYYY");
const existing = tp.file.find_tfile(fileName);
let workDailyNoteDisplay;
if (existing) {
  workDailyNoteDisplay = existing.basename;
} else {
  workDailyNoteDisplay = (await tp.file.create_new("#work #WorkDaily", fileName)).basename;
}
_%>

Work Daily Notes: [[<% workDailyNoteDisplay %>]]
```
